test/parallel/test-sqlite-session.js

```diff
@@ -3,9 +3,7 @@
 require('../common');
 const {
   DatabaseSync,
-  SQLITE_CHANGESET_OMIT,
-  SQLITE_CHANGESET_REPLACE,
-  SQLITE_CHANGESET_ABORT
+  constants,
```

Comment:
Nit, feel free to ignore:
```suggestion
  constants: {
    SQLITE_CHANGESET_OMIT,
    SQLITE_CHANGESET_REPLACE,
    SQLITE_CHANGESET_ABORT,
  },
```